configs/body/3d_kpt_mview_rgb_img/voxelpose/panoptic/prn64x64x64_cpn80x80x20_panoptic_cam5.py

```diff
@@ -0,0 +1,223 @@
+_base_ = ['../../../../_base_/datasets/panoptic_body3d.py']
```

Comment:
I suggest renaming the config file to `voxelpose_prn64x64x64-cpn80x80x20_panoptic.py` to meet the config naming conventions. Also please rename the corresponding .md file to `voxelpose_panotpic.md`.